requirements.txt

```diff
@@ -31,7 +31,7 @@ parsimonious==0.8.1
 py==1.10.0
 pyparsing==3.0.9
 pytest==6.2.0
-pytest-cov==2.10.1
+pytest-cov==3.0.0
```

Comment:
this should probably be in requirements-test.txt

Reply:
This might be used outside of tests too, I'll have to look into it. For these PRs, I am trying to avoid any change besides the version numbers.